src/opm/output/eclipse/Summary.cpp

```diff
@@ -1792,6 +1792,7 @@ static const std::unordered_map< std::string, ofun > funs = {
     { "GWCT", div( rate< rt::wat, producer >,
                    sum( rate< rt::wat, producer >, rate< rt::oil, producer > ) ) },
     { "WGOR", div( rate< rt::gas, producer >, rate< rt::oil, producer > ) },
+    { "WWGR", div( rate< rt::wat, producer >, rate< rt::gas, producer > ) },
```

Comment:
I think this will be zero for injectors regardless of injected phase, right?  If so, then this is fine.

Reply:
![image](https://user-images.githubusercontent.com/54812829/235084386-27518093-be90-49cd-b978-a9df70682d57.png)